discord/ext/bridge/bot.py

```diff
@@ -118,9 +118,10 @@ def decorator(func) -> BridgeCommandGroup:
         return decorator
 
     async def invoke(self, ctx: ExtContext | BridgeExtContext):
+        br_cmd = isinstance(ctx.command, BridgeExtCommand)
         if ctx.command is not None:
             self.dispatch("command", ctx)
```

Comment:
As long as we're sure that ctx.command isn't modified in this dispatch, this should be fine

Reply:
Shouldn't be an issue now, because I'm effectively checking for br_cmd every time it is needed.